src/routes/EventRouter.ts

```diff
@@ -127,8 +127,8 @@ export default class EventRouter {
 
       // calculate duration of flight
       const timeOfFlight = Calculations.calculateTimeOfFlight(
-        gameConfig.speed,
-        eventData.speed,
+        gameConfig.server.speed,
```

Comment:
Why the new server namespace?

Reply:
Because the game-speed did not fit elsewhere and maybe we will have more server-config-vars

Reply:
Note for me: Line 131 contains a bug

Reply:
^^ always useful these reviews ;)
Aye could, for example, provide server characteristics, like the language used and its limits...

Reply:
Resolved with commit ae528c2